src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```diff
@@ -45,14 +45,60 @@ tests:
               maxValue: 200
 
     - label: "TH reads from the DUT the (0xFFFC) FeatureMap attribute"
-      PICS: WNCV.S.Afffc
+      PICS:
+          " WNCV.S.Afffc && !WNCV.S.F00 && !WNCV.S.F01 && !WNCV.S.F02 &&
+          !WNCV.S.F03 && !WNCV.S.F04 "
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
+          value: 0
           constraints:
               type: bitmap32
-              minValue: 0
-              maxValue: 32768
+
+    - label: "Given WNCV.S.F00(LF) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F00
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label: "Given  WNCV.S.F01(TL) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F01
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    - label: "Given WNCV.S.F02(PA_LF) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F02
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x4]
```

Comment:
```suggestion
              hasMasksSet: [0x5]
```

Reply:
PA_LF must have LF set as well

Reply:
@jmeg-sfy We are checking each bit in a different step as different PICS are applicable. line no 58 checks for WNCV.S.F00(LF) flag and line no 76 checks for WNCV.S.F02(PA_LF)
![image](https://user-images.githubusercontent.com/102359958/187832395-3ab28f5f-9815-4edc-9fd6-fdbbf22e3fb5.png)